### PR TITLE
Feature/readme

### PR DESCRIPTION
**Pull request by vadymmarkov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,160 @@
...
+* [Contributing](#contributing)
+* [License](#license)
 
 ## Usage
 
+### Controller
+
+To start your slideshow just instantiate `LightboxController`, set needed
+delegates and present it:
+
```
Comment (RamonGilabert) [inline README.md]: 😱

File: README.md [hunk 2]
```diff
@@ -1,14 +1,160 @@
...
+  func lightboxControllerWillDismiss(_ controller: LightboxController) {
+    // ...
+  }
+}
+```
+
+### Image loading
+
+Bu default images are loaded using `sendAsynchronousRequest` method of
```
Comment (zenangst) [inline README.md]: @vadymmarkov what do you mean with `Bu` ? :)

File: README.md [hunk 3]
```diff
@@ -1,14 +1,160 @@
...
+  from.present(videoController, animated: true) {
+    videoController.player?.play()
+  }
+}
+```
+
+### Configuration
+
+Configure text, colors, fonts of UI elements by just overriding the static
+variables in the **Lightbox** [configuration](https://github.com/hyperoslo/Lightbox/blob/master/Source/LightboxConfig.swift) struct. As an example:
```
Comment (zenangst) [inline README.md]: I would change:

```
Configure text, colors, fonts of UI elements by just overriding the static
variables in the **Lightbox**
```

To:
```
Configure text, colors, fonts of UI elements by overriding the static
variables in the **Lightbox**
```

Just remove `just` (pun intended)

File: README.md [hunk 4]
```diff
@@ -1,14 +1,160 @@
...
+![Swift](https://img.shields.io/badge/%20in-swift%203.0-orange.svg)
+
+**Lightbox** is a convenient and easy to use image viewer for your iOS app,
+packed with all the features you expect:
+
+- [x] Paginated image slideshow.
+- [x] Video support.
+- [x] Double-tap to zoom.
+- [x] Image caption.
+- [x] Dynamic background.
```
Comment (zenangst) [inline README.md]: Should we describe what `Dynamic background` is? Or is that clear enough in itself?

**Comment by zenangst** [comment]
@vadymmarkov this is great!

File: README.md (hunk 4 above)
Comment (vadymmarkov) [inline README.md]: Could you write a few words about it so I will add? 😄

Comment (zenangst) [inline README.md]: We don't go into detail on anything else so we can just leave it 😁

**Comment by zenangst** [comment]
@vadymmarkov 🍰

**Pull request merged**